.github/workflows/knative-verify.yaml

```diff
@@ -53,7 +53,6 @@ jobs:
     - name: Check out code onto GOPATH
       uses: actions/checkout@v2
       with:
-        fetch-depth: 1
```

Comment:
why is this changing

Reply:
1 is default, ah well it will get rolled back. This PR is older than .github repo :D